packrat.js

```diff
@@ -230,14 +230,16 @@ Packrat.prototype.runCommand = function() {
         this.log('Packrat is running `%s`...', command);
     }
 
-    result = sh.exec(command, true);
+    result = spawnSync('sh', [ '-c', command ], {
+        stdio: [ 0, 1, 2 ]
+    });
 
     if (result.stdout) {
-        this.log(result.stdout);
+        this.log(String(result.stdout));
```

Comment:
`result.stdout` is `null` as you redirect spawned process stdout, so this `if` makes no sense.